core/templates/services/math-interactions.service.ts

```diff
@@ -61,9 +61,27 @@ export class MathInteractionsService {
         'not valid.';
     }
     if (errorMessage.indexOf('is not a valid postfix operator.') !== -1) {
+      let humanReadableOperator = (
+        errorMessage[0] === '*' ? '×' :
+        errorMessage[0] === '/' ? '÷' :
+        errorMessage[0]);
       errorMessage = (
-        'Your answer seems to be missing a variable/number after the "' +
-        errorMessage[0] + '".');
+        'Your answer seems to be missing a variable/number after the ' +
+        humanReadableOperator + ' operator.');
+    }
+    if (errorMessage === 'Not a prefix operator.') {
+      let i: number;
+      for (i = 0; i < expressionString.length; i++) {
+        if (expressionString[i] === '*' || expressionString[i] === '/') {
+          let humanReadableOperator = (
+            expressionString[i] === '*' ? '×' :
+            expressionString[i] === '/' ? '÷' :
+            null);
+          errorMessage = (
+            'Your answer seems to be missing a variable/number before the ' +
+            humanReadableOperator + ' operator.');
```

Comment:
Looks like we are overwriting this variable in a loop here. Given that it is an error message, can we just break out of the loop once an error is detected?

Reply:
@kevintab95 I think loop is required for the expressions like `*2/4*5(*3)/9`. Here two places are missing values(before `*2` and `*3`)

Reply:
Also @kevintab95 How to meet these conditions?
![image](https://github.com/oppia/oppia/assets/91023072/5fbb4c77-962c-4706-a5ec-00b622e7d75d)

Reply:
> @kevintab95 I think loop is required for the expressions like `*2/4*5(*3)/9`. Here two places are missing values(before `*2` and `*3`)

Hm, given that we just show one error message, should we just exit the loop and show that error?

-------

For the comment in the screenshot, I am not sure I have the full context, can you provide a link to the comment please? Also, it would help if you can specify which part of the comment needs further clarity. Thanks!

Reply:
@kevintab95 here is the link for that comment https://github.com/oppia/oppia/pull/16969#:~:text=We%20should%20try%20to%20give%20the%20most%20specific
Also, I have no confusion in the comment what to do as asked by Sean, But how to approach that I am not getting.